openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -2312,6 +2340,11 @@ The technology described in this specification was made available from contribut
 
    [[ To be removed from the final specification ]]
 
+   -15
+
+   * credential response always returns an array when not returning a transaction_id with the option for additional meta-data
+   * deferred credential response always returns an array (same as credential response)
+
```

Comment:
Until d14, `notification_id` was associated to a single credential.

I think that readers/implementers would appreciate to highlight the change on the granularity of `notification_id` (and notification events)

Reply:
That is a good point, I will add an explicit point to the document history